src/js/base/fetch.js

```diff
@@ -169,7 +169,7 @@ export default async(url, options) => {
           }
         }
 
-        if (response.status >= 500) {
+        if (response.status >= 500 || !response.status) {
           Radio.trigger('event-router', 'unknownError', response.status);
```

Comment:
Given that the previous line MAY pass `response.status: undefined` will the receiver of radio be able to handle it? Should this be type cast?

Reply:
`response.status` is a javascript api object and will never be non-numeric